scenariolib/uabot.go

```diff
@@ -32,6 +32,7 @@ type uabot struct {
 // NewUabot will start a bot to run some scenarios. It needs the url/path where to find the scenarions {scenarioURL},
 // the searchToken, the analyticsToken and a randomizer.
 func NewUabot(local bool, scenarioURL string, searchToken string, analyticsToken string, random *rand.Rand) Uabot {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Makes more sense to me to have this `Seed()` call in main.go when random is defined.

Reply:
Yea the point is that the main can define a seed if need be, that's why the bot receives a *rand.Rand as parameter.

Could receive a seed instead if you think it makes more sense.